R/plotFMM.R

```diff
@@ -1,12 +1,53 @@
-# Plot fitted FMM models
-#
-# Arguments:
-#   objFMM: object of class FMM.
-#   components: TRUE to plot each component separately.
-#   plotAlongPeriods: TRUE to plot more than 1 period.
-#   use_ggplot2: TRUE to use ggplot2 package.
-#   legendInComponentsPlot: TRUE to indicate if the a legend should be plotted in the component plot.
-#   textExtra: Extra text to be added to the titles of the plot.
+#' Plot fitted FMM models
+#'
+#' \code{plotFMM()} is used to plot fitted FMM models. The function can either
+#' plot the fitted model against the data or each of the components of the model
+#' separately. Optionally \code{'ggplot2'} can be used as graphic library.
+#'
+#' @param objFMM Object of class FMM
```

Comment:
To create the final autogenerated Rd files, the information previously present in the .R files should be combined with the one available in the previous versions of Rd files (the parameters of the function are more detailed in the previous Rd files)